chore: remove aioch as a dep

hazzadous opened the pull request:
We're not using an asyncio event loop as far as I can tell, but maybe
I'm missing something.

## Problem

<!-- Who are we building for, what are their needs, why is this important? -->

## Changes

<!-- If there are frontend changes, please include screenshots. -->
<!-- If a reference design was involved, include a link to the relevant Figma frame! -->

👉 *Stay up-to-date with [PostHog coding conventions](https://posthog.com/docs/contribute/coding-conventions) for a smoother review.*

## How did you test this code?

<!-- Briefly describe the steps you took. -->
<!-- Include automated tests if possible, otherwise describe the manual testing routine. -->

Commit:
chore: remove aioch as a dep

We're not using an asyncio event loop as far as I can tell, but maybe
I'm missing something.

Commit:
fix lint

Twixes reviewed (commented):
I don't believe we use this, but I think @fuziontech will have the most context, as he wrote this originally =.

Twixes reviewed (commented):
I don't believe we use this, but I think @fuziontech will have the most context, as he wrote this originally.

Commit:
use python 3.8

Commit:
Merge remote-tracking branch 'origin/master' into chore/remove-aioch

guidoiaquinti reviewed (approved):
VAI FORTE

Commit:
ci: ensure we have xmlsec1 installed in backend CI

If we do not, and we miss the virtualenv cache then we are unable to
install the pip requirements.
